docs/instructions_nuimages.md

```diff
@@ -96,23 +121,41 @@ nuImages includes surface classes as well:
 | [`vehicle.ego`](#2-vehicleego) | The vehicle on which the cameras, radar and lidar are mounted, that is sometimes visible at the bottom of the image. |
 
 ### 1. flat.driveable_surface
-[### TODO add examples images of class]
+![driveable_1](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/driveable_1.png)
+![driveable_2](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/driveable_2.png)
+![driveable_3](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/driveable_3.png)
+![driveable_4](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/driveable_4.png)
+
 ### 2. vehicle.ego
-[### TODO add examples images of class]
+![ego_1](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/ego_1.png)
+![ego_2](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/ego_2.png)
+![ego_3](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/ego_3.png)
+![ego_4](https://www.nuscenes.org/public/images/taxonomy_imgs/nuimages/ego_4.png)
 
 ## Semantic Segmentation
 ### General Instructions
  - Only annotate a surface if its length and width are **both** greater than 20 pixels.
  - Annotations should tightly bound the edges of the area(s) of interest. 
+ 
+ [![](https://s3.amazonaws.com/nutonomy-imdb-dummy/semantic_seg_examples/new_endpoint/no_gaps.png)]()
  - If two areas/objects of interest are adjacent to each other, there should be no gap between the two annotations.
+ 
+ [![](https://s3.amazonaws.com/nutonomy-imdb-dummy/semantic_seg_examples/new_endpoint/polygon_tight.png)]()
  - Annotate a surface only as far as it is clearly visible.
+ 
+ [![](https://s3.amazonaws.com/nutonomy-imdb-dummy/semantic_seg_examples/ground/rules1.png)]()
  - If a surface is occluded (e.g. by branches, trees, fence poles), only annotate the visible areas (which are more than 20 pixels in length and width).
+ 
+ [![](https://camo.githubusercontent.com/dfbfa4c8a61590f81761ba1e7b47d68fe5d92773/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6e75746f6e6f6d792d696d64622d64756d6d792f73656d616e7469635f7365675f6578616d706c65732f6e65775f656e64706f696e742f6275736865732e706e67)]()
```

Comment:
This is the wrong link. That's the temporary copy on Github which may change any time.